opa/views/session_views.xml

```diff
@@ -130,4 +130,49 @@
         <field name="res_model">session</field>
         <field name="view_mode">tree,kanban,calendar,gantt,graph,form</field>
     </record>
+
+    <record id="action_graph_session" model="ir.actions.act_window">
+        <field name="name">Attendees by Course</field>
+        <field name="res_model">session</field>
+        <field name="view_mode">graph</field>
+        <field name="view_id" ref="session_graph_view"/>
+    </record>
+
+    <record id="action_calendar_session" model="ir.actions.act_window">
+        <field name="name">Sessions' Calendars</field>
```

Comment:
"Schedule of sessions" or "Calendar of Sessions"

Reply:
I'll change it.